app/const.js

```diff
@@ -1,58 +1,74 @@
 export const SESSIONSCHEMA = {
-    "type": "jsonschema",
-    "schema": {
-        "type": "object",
-        "properties": {
-            "profileId": {
-                "id": "profileId",
-                "type": "string",
-                "pattern": "\\w+\\.\\w+"
-            },
-            "profileVersion": {
-                "id": "profileVersion",
-                "type": "string"
-            },
-            "experimentId": {
-                "id": "experimentId",
-                "type": "string",
-                "pattern": "\\w+\\.\\w+\\.\\w+"
-            },
-            "experimentVersion": {
-                "id": "experimentVersion",
-                "type": "string"
-            },
-            "sequence": {
-                "id": "sequence",
-                "type": "array",
-                "items": {
-                    "type": "string"
+    'type': 'jsonschema',
+    'schema': {
+        'type': 'object',
+        'properties': {
+            'profileId': {
+                'id': 'profileId',
+                'type': 'string',
+                'pattern': '\\w+\\.\\w+'
+            },
+            'profileVersion': {
+                'id': 'profileVersion',
+                'type': 'string'
+            },
+            'experimentId': {
+                'id': 'experimentId',
+                'type': 'string',
+                'pattern': '\\w+\\.\\w+\\.\\w+'
+            },
+            'experimentVersion': {
+                'id': 'experimentVersion',
+                'type': 'string'
+            },
+            'sequence': {
+                'id': 'sequence',
+                'type': 'array',
+                'items': {
+                    'type': 'string'
                 }
             },
-            "softwareVersion": {
-                "id": "softwareVersion",
-                "type": "string"
+            'softwareVersion': {
+                'id': 'softwareVersion',
+                'type': 'string'
             },
-            "expData": {
-                "id": "expData",
-                "type": "object"
+            'expData': {
+                'id': 'expData',
+                'type': 'object'
             },
-            "feedback": {
-                "id": "feedback",
-                "type": "string"
+            'feedback': {
+                'id': 'feedback',
+                'type': 'string'
             },
-            "hasReadFeedback": {
-                "id": "hasReadFeedback",
-                "type": "boolean"
+            'hasReadFeedback': {
+                'id': 'hasReadFeedback',
+                'type': 'boolean'
+            },
+            'didComplete': {
```

Comment:
See discussion on parallel `didComplete` type implementation here:
https://github.com/CenterForOpenScience/exp-addons/pull/50#issuecomment-194641398

(realized I needed it while working on changes to session serialization)

If we do choose this version of the field name, note that the field name shown here does not match the ID in the line below.